src/eslint-config-adeira/__tests__/__snapshots__/index.test.js.snap

```diff
@@ -715,6 +717,7 @@ Object {
     "react/jsx-uses-vars": 2,
     "react/jsx-wrap-multilines": "off",
     "react/no-access-state-in-setstate": 2,
+    "react/no-adjacent-inline-elements": 0,
```

Comment:
I would maybe consider using this rule too

Reply:
I don't think we need it since doing 

```
const test = (<div><a></a><a></a></div>)
```
will be formatted to 

```
(
    <div>
      <a />
      <a />
    </div>
  )
```

Reply:
By prettier